lib/foreman_remote_execution/engine.rb

```diff
@@ -27,7 +29,7 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_remote_execution.register_plugin', after: :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_remote_execution do
-        requires_foreman '>= 1.10'
+        requires_foreman '>= 1.11'
```

Comment:
I fail to see the backward incompatibility on the line 17: I see the `+` operator is not defined on the new structure (as it used to be an array before), but `<<` should work on both